.github/dependabot.yml

```diff
@@ -64,6 +62,32 @@ updates:
       - dependency-name: github.com/elastic/mito
     open-pull-requests-limit: 2
 
+  - package-ecosystem: gomod
+    directory: /
+    # Workaround to satisfy Dependabot's unique constraint (needs unique 'package-ecosystem', 'directory', 'target-branch' tuples) 
+    # for duplicate package-ecosystem configurations: https://github.com/dependabot/dependabot-core/issues/1778#issuecomment-1988140219
+    target-branch: main
+    schedule:
+      interval: daily
```

Comment:
@shmsr : Do we need daily runs ? 
Or weekly is fine ?

Reply:
is the reason that we need to duplicate the entire section because we want our own `cooldown` configuration?